fighthealthinsurance/staff_views.py

```
@@ -453,3 +466,147 @@ def _chooser_stats(
+                return self._render_record(
+                    request,
+                    pro,
+                    draft=body,
+                    subject=subject,
+                    skip_reason=skip_reason,
+                    error=f"{pro.email} is not a sendable address; cannot send.",
+                    status=400,
+                )
+            try:
+                send_partner_intro_email(pro, subject=subject, body=body)
+            except Exception as e:
```

Ruff (0.15.17)

[warning] Do not catch blind exception: `Exception`

(BLE001)